earth_enterprise/src/fusion/portableglobe/servers/local/preview/setup.html

```diff
@@ -44,9 +44,7 @@
   <div id="earth"></div>
   <div id="NoMap" style="display: none;">
     <div class="GlobeImgText">
-      <span>There was a problem loading the Google Earth Plugin.</span>
-      <span>The Google Earth Plugin is currently only available on <b>Windows</b> and <b>Mac OS X 10.6+</b>.</span>
-      <span><a href="http://earth.google.com/intl/en/plugin/">Click here to learn more.</a></span>
+      <span>3D previews are not available, please use the <b>GEEC</b> to view your globe.</span>
```

Comment:
I might change the wording slightly, like instead of "not available", maybe specify "not available in a web browser".
And I think it might be good to spell out "Google Earth Enterprise Client" instead of "GEEC".
Otherwise, I think that's a good message.